tests/mlir_printer_test.py

```diff
@@ -73,7 +73,7 @@ class ParamAttrWithCustomFormat(ParametrizedAttribute):
     param1: ParameterDef[ParamAttr]
 
     def print_parameters(self, printer: Printer) -> None:
-        printer.print(f"~~")
+        printer.print(f'{"~~"}')
```

Comment:
Why changing this this way?
This is now different, and also this should make the test fail.
However since this test is not ran (since it is in the `mlir` folder), it is not shown on the CI.
I would replace it by `printer.print("~~")` then

Reply:
Are you sure it is not printing the same thing?

Reply:
I am quite sure this test is run as part of the pytest execution, so this should run on the CI. However, I would go with Mathieu's suggestions as I don't see a reason here to use a formatted string.

Reply:
I formatted towards being fine with flake8 linter. I can remove if you want

Reply:
I think `printer.print("~~")` is best, so without the format string.